<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>founding_cook</class>
 <widget class="QDialog" name="founding_cook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <item>
    <widget class="QFrame" name="frame">
     <property name="outer_stylesheet" stdset="0">
      <string>frame_detail_show</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1,0,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>开台单号:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_ch_billno"/>
        </item>
        <item>
         <widget class="pushbutton_underline" name="pushButton_inline">
          <property name="text">
           <string>网络订单导入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="pushbutton_underline" name="pushButton_div">
          <property name="text">
           <string>分单</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3" rowspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_div"/>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>人数:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="lds_spinbox" name="lineEdit_int_person"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_vch_operid"/>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>时间:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="lds_datetimeEdit" name="dateTimeEdit_dt_operdate"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>服务员:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>开台人:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit_vch_waiter"/>
        </item>
        <item>
         <widget class="pushbutton_underline" name="pushButton_9">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>备注信息:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_vch_memo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_stack1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_bar_clear">
        <property name="text">
         <string>清台</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_13">
        <property name="text">
         <string>点菜</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_open">
        <property name="text">
         <string>开台</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_14">
        <property name="text">
         <string>退出</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_stack2">
     <layout class="QHBoxLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_reprint">
        <property name="text">
         <string>预打单</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_pay">
        <property name="text">
         <string>结算</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_pay_div">
        <property name="text">
         <string>分开结算</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_ed_order">
        <property name="text">
         <string>加菜</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_ed_save">
        <property name="text">
         <string>保存</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_toolbar_button" name="pushButton_ed_exit">
        <property name="text">
         <string>退出</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_spinbox</class>
   <extends>QSpinBox</extends>
   <header>lds_spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_datetimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>lds_datetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>pushbutton_underline</class>
   <extends>QPushButton</extends>
   <header>pushbutton_underline.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_13</tabstop>
  <tabstop>pushButton_ed_order</tabstop>
  <tabstop>pushButton_ed_exit</tabstop>
  <tabstop>lineEdit_vch_waiter</tabstop>
  <tabstop>dateTimeEdit_dt_operdate</tabstop>
  <tabstop>lineEdit_vch_operid</tabstop>
  <tabstop>lineEdit_vch_memo</tabstop>
  <tabstop>pushButton_reprint</tabstop>
  <tabstop>pushButton_pay</tabstop>
  <tabstop>pushButton_pay_div</tabstop>
  <tabstop>lineEdit_ch_billno</tabstop>
  <tabstop>pushButton_bar_clear</tabstop>
  <tabstop>pushButton_div</tabstop>
  <tabstop>pushButton_open</tabstop>
  <tabstop>pushButton_14</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_inline</tabstop>
  <tabstop>pushButton_ed_save</tabstop>
  <tabstop>lineEdit_int_person</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
